pkgs/misc/drivers/infnoise/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchFromGitHub, git, libftdi }:
+
+stdenv.mkDerivation rec {
+  name = "infnoise";
+
+  src = fetchFromGitHub {
+    owner = "13-37-org";
+    repo = "${name}";
+    rev = "132683d4b5ce0902468b666cba63baea36e97f0c";
+    sha256 = "0r8kk8zwy4a4mwfwbq68hq23w88x5v2sf4zkyjizv9kb943mxs08";
+    leaveDotGit = true;
```

Comment:
Why is this required?

Reply:
During the build it gets info from the git repo to embed into the binary what commit it comes from, using define's.

Reply:
Can we patch that to use version instead? Leaving dotGit is not fully reproducible.

Reply:
Ah, that's unfortunate...